lib/rubocop/cop/rails/logger_block.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks that blocks are passed to Rails.logger instead of values.
+      #
+      # @example
+      #   #bad
+      #   Rails.logger.debug 'Beginning import.'
+      #
+      #   #good
+      #   Rails.logger.debug { 'Beginning import.' }
+      #
+      class LoggerBlock < Base
```

Comment:
What about `Rails/EagerEvaluationLogMessage` as a name that represents the problem?

```suggestion
      class EagerEvaluationLogMessage < Base
```

Cc @rubocop/rubocop-core

Reply:
I agree with this, I am using your suggestion for now unless someone has a better suggestion later.